Code/autopkg

```diff
@@ -2365,7 +2353,9 @@ def run_recipes(argv):
                 # record any summary results
                 output_keys = item["Output"].keys()
                 results_keys = [
-                    key for key in output_keys if key.endswith("_summary_result")
+                    summary_key
+                    for summary_key in output_keys
+                    if key.endswith("_summary_result")
```

Comment:
Should this not be `summary_key` also?

Reply:
Yep, looks like it to me.

Reply:
I thought Nick was just reformatting and moving functions to other modules. Sigh

Reply:
Resolved by https://github.com/autopkg/autopkg/pull/531/files